build/golint.sh

```diff
@@ -22,7 +22,9 @@ TIMEOUT="10m"
 
 echo "Running golangci-lint..."
 
-golangci-lint run --timeout ${TIMEOUT} --skip-dirs ${SKIP_DIR_REGEX} -E maligned,whitespace,gocognit,unparam -e '`ctx` is unused'
+# https://github.com/mdempsky/maligned is deprecated.
+# maligned | The repository of the linter has been archived by the owner. Replaced by govet 'fieldalignment'.
+golangci-lint run --timeout ${TIMEOUT} --skip-dirs ${SKIP_DIR_REGEX} -E govet,whitespace,gocognit,unparam -e '`ctx` is unused'
```

Comment:
(Un?)related question, not requesting a change. Why isn't a configuration file used for these runs?

Reply:
there is https://github.com/kanisterio/kanister/pull/1921 this PR which adds configuration file. But not sure about what's pending there.